src/Generics/Linear/TH/Insertions.hs

```diff
@@ -0,0 +1,41 @@
+{-# language LinearTypes #-}
+{-# language MagicHash #-}
+{-# language NoImplicitPrelude #-}
+{-# language PolyKinds #-}
+{-# language TypeOperators #-}
+{-# options_haddock hide #-}
+module Generics.Linear.TH.Insertions where
+import qualified Generics.Linear.Class as G
+import qualified GHC.Exts as E
+
+infixr 9 .
+(.) :: (b %m-> c) -> (a %m-> b) -> a %m-> c
```

Comment:
Does this typecheck? Did we add a `MulMult m m = m` rule? If we did this will probably come and bite us in the future.